pkg/cli/export/export_config.go

```diff
@@ -0,0 +1,114 @@
+package export
+
+import (
+	"bytes"
+	"encoding/json"
+
+	lib_certifications "github.com/opencontrol/compliance-masonry/pkg/lib/certifications"
+	"github.com/opencontrol/compliance-masonry/pkg/lib/common"
+)
+
+////////////////////////////////////////////////////////////////////////
+// Package structures
+
+// Config contains settings for this object
+type Config struct {
+	// remainder are configuration settings local to Export
+	Certification   string
+	OpencontrolDir  string
+	DestinationFile string
+	OutputFormat    OutputFormat
+	Flatten         bool
+	InferKeys       bool
+	Docxtemplater   bool
+	KeySeparator    string
+}
+
+// internal - structure for JSON / YAML output
+type exportData struct {
+	Certification common.Certification
+	Components    []common.Component
+	Standards     []common.Standard
+}
+
+// MarshalJSON provides JSON support
+func (p *exportData) MarshalJSON() (b []byte, e error) {
+	// start the output
+	buffer := bytes.NewBufferString("{")
+
+	// certification
+	buffer.WriteString("\"certification\":")
+	bytesJSON, err := lib_certifications.MarshalJSON(p.Certification)
+	if err != nil {
+		return nil, err
+	}
+	buffer.WriteString(string(bytesJSON))
+
+	// iterate over components
```

Comment:
``components`` and ``standard`` iterations are very similar. Could this be condensed into a for each loop?

e.g.
`````
    arrayOpenControl := [2]string{"components", "standards"}

    for index,element := range arrayOpenControl{
        if len(p.element) > 0 {
           .... do stuff
        }     
    }   
}
`````

Reply:
@shawndwells this actually would probably be a separate function as it deals with more than just strings.
I was hoping to avoid code cleanup in these type of PRs because of how big they are.

Reply:
Tracked in #295

Reply:
Sounds good. Thanks for opening the ticket to track.